docs/architecture/modern-web-apps-azure/work-with-data-in-asp-net-core-apps.md

```diff
@@ -194,9 +194,9 @@ private void ConfigureOrder(EntityTypeBuilder<Order> builder)
 }
 ```
 
-In this example, the `ShipToAddress` property is of type `Address`. `Address` is a value object with several properties such as `Street` and `City`. EF Core maps the `Order` object to its table with one column per `Address` property, prefixing each column name with the name of the property. In this example, the `Order` table would include columns such as `ShipToAddress_Street` and `ShipToAddress_City`.
+In this example, the `ShipToAddress` property is of type `Address`. `Address` is a value object with several properties such as `Street` and `City`. EF Core maps the `Order` object to its table with one column per `Address` property, prefixing each column name with the name of the property. In this example, the `Order` table would include columns such as `ShipToAddress_Street` and `ShipToAddress_City`. It's also possible to store owned types in separate tables, if desired.
 
-[EF Core 2.2 introduces support for collections of owned entities](https://docs.microsoft.com/ef/core/what-is-new/ef-core-2.2#collections-of-owned-entities)
+Learn more about owned [entity support in EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/owned-entities).
```

Comment:
```suggestion
Learn more about owned [entity support in EF Core](https://docs.microsoft.com/ef/core/modeling/owned-entities).
```

Reply:
I changed your link to have them supported in VS offline books too across books